homeassistant/components/blink/camera.py

```diff
@@ -8,24 +8,51 @@
 from typing import Any
 
 from requests.exceptions import ChunkedEncodingError
+import voluptuous as vol
 
 from homeassistant.components.camera import Camera
 from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import ATTR_ENTITY_ID, CONF_FILE_PATH, CONF_FILENAME
 from homeassistant.core import HomeAssistant
-from homeassistant.exceptions import HomeAssistantError
+from homeassistant.exceptions import HomeAssistantError, ServiceValidationError
 from homeassistant.helpers import entity_platform
+import homeassistant.helpers.config_validation as cv
 from homeassistant.helpers.device_registry import DeviceInfo
 from homeassistant.helpers.entity_platform import AddEntitiesCallback
 from homeassistant.helpers.update_coordinator import CoordinatorEntity
 
-from .const import DEFAULT_BRAND, DOMAIN, SERVICE_TRIGGER
+from .const import (
+    DEFAULT_BRAND,
+    DOMAIN,
+    SERVICE_REFRESH,
+    SERVICE_SAVE_RECENT_CLIPS,
+    SERVICE_SAVE_VIDEO,
+    SERVICE_TRIGGER,
+)
 from .coordinator import BlinkUpdateCoordinator
 
 _LOGGER = logging.getLogger(__name__)
 
 ATTR_VIDEO_CLIP = "video"
 ATTR_IMAGE = "image"
 PARALLEL_UPDATES = 1
+SERVICE_SAVE_VIDEO_SCHEMA = vol.Schema(
```

Comment:
Please i line the schema in the register call and just pass a dict. Don't include the entity_id.